receiver/sqlqueryreceiver/logs_receiver.go

```diff
@@ -174,13 +174,13 @@ func (receiver *logsReceiver) Shutdown(ctx context.Context) error {
 		return nil
 	}
 
+	var errors error
 	receiver.settings.Logger.Debug("stopping...")
 	receiver.stopCollecting()
 	for _, queryReceiver := range receiver.queryReceivers {
-		queryReceiver.shutdown(ctx)
+		errors = multierr.Append(errors, queryReceiver.shutdown(ctx))
```

Comment:
Lets not use `multierror`, we recently had a PR to remove its use: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31768

Reply:
NVM, that was the internal package.

Reply:
Good call, I've updated.

Reply:
I've updated to use `errors.Join`, I'm fine with leaving it in (even though the scope of this PR is pretty wide now), or removing it. Just let me know.